tests/e2e/specs/edit-posts.test.js

```diff
@@ -53,24 +49,24 @@ test.describe( 'Edit Posts', () => {
 		await editor.publishPost();
 		await admin.visitAdminPage( '/edit.php' );
 
-		await page.waitForSelector( '#the-list .type-post' );
+		const listTable = page.getByRole( 'table', { name: 'Table ordered by' } );
+		await expect( listTable ).toBeVisible();
 
 		// Click the post title (edit) link
-		await page.locator( `//a[contains(@class, "row-title")][contains(text(), "${ title }")]` ).click();
+		await listTable.getByRole( 'link', { name: `“${ title }” (Edit)` } ).click();
```

Comment:
If we care about an exact match, we need to explicitly set [the `exact` option](https://playwright.dev/docs/api/class-framelocator#frame-locator-get-by-role-option-exact) to `true` as it's `false` by default. If we're OK with a partial match, the `name: title` should be sufficient here!

Reply:
A few other quick action links in the table row contain titles, and the test needs to click on the edit link. With suffix, there's no need to exact match.